src/python/pants/backend/jvm/subsystems/rsc_native_image.py

```diff
@@ -0,0 +1,31 @@
+# coding=utf-8
+# Copyright 2019 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+from pants.backend.jvm.subsystems.zinc import _ZINC_COMPILER_VERSION
+from pants.binaries.binary_tool import NativeTool
+from pants.util.memo import memoized_method
+
+
+class RscNativeImage(NativeTool):
```

Comment:
Should probably rename the subsystem to just `Rsc`, and the file to just `rsc.py` (and the BUILD file reference).